Exiled.API/Features/Door.cs

```diff
@@ -516,11 +539,18 @@ public void Lock(float time, DoorLockType lockType)
         /// <param name="flagsToUnlock">The <see cref="Enums.DoorLockType"/> of the lockdown.</param>
         public void Unlock(float time, DoorLockType flagsToUnlock) => DoorScheduledUnlocker.UnlockLater(Base, time, (DoorLockReason)flagsToUnlock);
 
+        /// <summary>
+        /// Checks if specified <see cref="Player"/> can interact with the door.
+        /// </summary>
+        /// <param name="player">Player to check.</param>
+        /// <returns><see langword="true"/> if anybody can interact. Otherwise, false.</returns>
```

Comment:
```suggestion
        /// <returns><see langword="true"/> if the specified player can interact with the door. Otherwise, <see langword="false"/>.</returns>
```